Automattic/woocommerce-payments
# Update Payments Overview deposits UI to simplify how we communicate upcoming deposits

Fixes #7646

#### Changes proposed in this Pull Request

This PR introduced the updates **Payments Overview → Deposits UI** to simplify how we communicate upcoming deposits to merchants. See design on [Figma](https://www.figma.com/file/aDFNTeogq2cXSr4Nqeni8g/%5BWCPay%5D-Instant-Deposits?node-id=306:30781&mode=dev).

This new UI design removes the `Next deposit` table and replaces it with an informative deposit schedule message tailored to the merchant's schedule. Notice banners that were previously rendered in the `Next deposit` section are now rendered underneath the deposit schedule message.

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/db5b2688-ab3a-4cce-8775-2e133a13c2c5">

#### TODO
- [x] Figure out how to determine a condition for rendering loan notice.
	- Handled in issue #7689
- [x] Improve loading state
- [x] Update schedule message with new simplified version
- [x] Confirm UI for available balance under minimum amount
	- Handled in issue #6774
- [x] Testing instructions
- [x] Screenshot of each case
- [x] Test on mobile


#### Screenshots
<details open>
<summary>Toggle screenshots</summary>

#### ✅ Next deposit expected: positive available balance above minimum deposit threshold 👇

**Daily schedule**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/db5b2688-ab3a-4cce-8775-2e133a13c2c5">

**Weekly schedule**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/169af7ae-a5d2-4c42-a28f-b831ce7bd7c4">

**Monthly schedule**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/95ace66d-c9f8-4572-be55-dae2c8a2c4fe">

**Monthly schedule: last day of the month**
<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/75204f08-a319-4956-ad47-4d82994fda5c">



#### ❌ No next deposit expected 👇

**New account, no transactions** – No deposit UI rendered.

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/a3735075-4324-49d3-9ef6-23357c85760b">

**0 available balance, positive pending balance** – No deposit UI rendered.

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/0c79e4da-4524-486e-91e0-925164b4ffdf">

**Positive available balance < minimum deposit amount** – Deposit UI rendered as expected

✏️ This will be addressed in issue #6774. See [design](https://www.figma.com/file/aDFNTeogq2cXSr4Nqeni8g/%5BWCPay%5D-Instant-Deposits?type=design&node-id=294-83778&mode=design&t=exSeHwn5tRxr33zg-4).

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/ee9c40ef-339e-4af7-bf63-0c556f218ed6">



**Negative available balance** – Deposit UI rendered with notice.

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/46a90c21-619c-462d-940f-7b901817c2e8">

**Deposits blocked** – Deposit UI rendered with notice and no "change schedule" button.

<img width="400" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/c33467dd-2f3f-4459-9e60-e4132e210ea6">


#### How this works for multiple deposit currencies

[Tested on the Helix WooExpress demo site.](https://woo-delectably-sparkly-ninja.wpcomstaging.com/wp-admin/admin.php?page=wc-admin&path=/payments/overview&selected_currency=nok)

https://github.com/Automattic/woocommerce-payments/assets/3147296/c8d13038-7c4c-49f0-99c3-08974d36f0ba


#### Example on mobile viewport

<img width="375" alt="image" src="https://github.com/Automattic/woocommerce-payments/assets/3147296/f9774dbc-a8e4-48e8-aa63-c77274d50305">


</details>

#### Testing instructions





Check the new deposits UI on the **Payments → Overview** screen for each of the following cases and ensure it is displayed as expected. See the screenshots above as a reference for the new UI for each case.

* New account, no transactions
* 0 available balance, positive pending balance
* Positive available balance less than the minimum deposit threshold (e.g. $5 USD)
* Positive available balance greater than the minimum deposit threshold (e.g. $5 USD)
	* Daily schedule
	* Weekly schedule
	* Monthly schedule
	* Manual schedule
* Negative available balance
* Deposits blocked

-------------------


**Post merge**
